packages/message-system/src/config/config.v1.json

```diff
@@ -1,21 +1,51 @@
 {
     "version": 1,
     "timestamp": "2023-01-05T00:00:00+00:00",
-    "sequence": 21,
+    "sequence": 22,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=23.2.0",
+                        "mobile": "!",
+                        "web": ">=23.2.0"
+                    }
+                }
+            ],
+            "message": {
+                "id": "c4196202-b796-4c67-abcb-ac38c15ae343",
+                "priority": 90,
+                "dismissible": false,
+                "variant": "info",
+                "category": ["feature"],
+                "content": {
+                    "en-GB": "coinjoin config averageAnonymityGainPerRound",
+                    "en": "coinjoin config averageAnonymityGainPerRound",
+                    "es": "coinjoin config averageAnonymityGainPerRound",
+                    "cs": "coinjoin config averageAnonymityGainPerRound",
+                    "ru": "coinjoin config averageAnonymityGainPerRound",
+                    "ja": "coinjoin config averageAnonymityGainPerRound"
+                },
+                "feature": [
+                    { "domain": "coinjoin", "flag": true, "averageAnonymityGainPerRound": 5 }
+                ],
+                "context": { "domain": "accounts.coinjoin" }
+            }
+        },
         {
             "conditions": [
                 {
                     "environment": {
                         "desktop": "<23.2.0",
                         "mobile": "!",
-                        "web": "!"
+                        "web": "<23.2.0"
```

Comment:
The content should be adjusted a bit as well I suppose. `To use coinjoin, update to the latest version of the  **desktop** app.`

Reply:
on web it's available only locally in dev build so I guess it doesn't really matter